tutorials/hello/hello_client/test/unit/modules/hello/commands/__snapshots__/create_hello_command.spec.ts.snap

```diff
@@ -0,0 +1,20 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CreateHelloCommand constructor should have valid schema 1`] = `
+{
+  "$id": "hello/createHello-params",
```

Comment:
```suggestion
  "$id": "hello/createHelloParams",
```
or you can also do something like `"$id": "hello/createHelloCommand/params"`

Reply:
this is system-generated! I haven't changed it.